common/ambassador/base/deployment.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: apps/v1beta1
 kind: Deployment
 metadata:
   name: ambassador
+  namespace: istio-system
```

Comment:
If we didn't specify namespace, how did it end up in namespace kubeflow?
I mean, shouldn't there be a way to set namespace for all components

Reply:
Currently kfctl code use kubeflow ns by default